Removes MediHound's Noncon Vore

CameronWoof opened the pull request:
:cl: CameronWoof
tweak: The MediHound's now has a unique preference, which defaults to on.
add: MediHound now has a cyborg hypospray, to help tend to patients with their sleeper preference off.
/:cl:

Aaand we're back! MediHounds will have a few less patients for their sleeper, but they've been compensated with a cyborg hypospray. The preference defaults to 'allow', making it strictly opt-out.

CameronWoof committed:
Removes MediHound's Noncon Vore

MediHound once again respects vore preferences, as intended.

To make up for this, they now receive a unique cyborg hypospray with fewer reagents, increased energy costs, and no ability to upgrade.

CameronWoof committed:
Moves Preference Check to New Sleeper-Specific Variable, Buffs Hypo

Rather than using target.devourable, MediHounds now check for a new target.sleeperable preference, which defaults to TRUE.

Also, removed the custom HoundHypo. MediHounds will now get the full borg hypospray.

Poojawa commented on code/citadel/dogborgs.dm:
```diff
@@ -70,6 +70,7 @@
 	modules += new /obj/item/soap/tongue(src)
 	modules += new /obj/item/device/healthanalyzer(src)
 	modules += new /obj/item/dogborg/sleeper(src)
+	modules += new /obj/item/reagent_containers/borghypo(src)
```
This file is more or less kept as a reference to 5th port. since stuff's modularized out, you don't need to add code to this.

Poojawa commented on code/modules/vore/eating/vorepanel_vr.dm:
```diff
@@ -196,6 +196,12 @@
 		if(FALSE)
 			dat += "<a href='?src=\ref[src];toggledvor=1'>Toggle Devourable (Currently: OFF)</a>"
 
+	switch(user.sleeperable)
```
Putting this in the vore panel basically means that they still have to look at the settings, it's okay to add this to modules/client/preferences and savefile. just look at 'window flashing' or 'lobby music' or some other yes/no choice for hints on how to implement it.

CameronWoof committed:
Removes unnecessary 5th port code

CameronWoof committed:
Breaks the patch for Bhijn's viewing pleasure

DNM etc

CameronWoof committed:
Still not fixed

CameronWoof committed:
Fixes it again.

MediHound sleeper preference is now a game preference toggle, rather than a vore panel toggle.

MediHounds will not be able to ingest SSD or catatonic players.

Poojawa committed:
Merge branch 'master' into nomorenoncon
